### PR TITLE
Address management

### PR DESCRIPTION
**Pull request by BelideNikhil**
## Changes

+ Address Management
+ Checkout page
+ Apply Coupon at checkout

### CODE REVIEW

**Review by anuj-kumary — APPROVED** [review]
- Functionality Working as expected 
- Clean code and good use of naming conventions
Good to merge

File: src/Components/ApplyCoupon/ApplyCoupon.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                                }`}
+                                key={item.id}
+                            >
+                                <input
+                                    type="radio"
+                                    className="form-input mr-8"
+                                    name="coupon"
+                                    value={item.discount}
+                                    checked={couponValue === item.discount}
+                                    onChange={(e) => setCouponValue(Number(e.target.value))}
```
Comment (tanveertkd) [inline src/Components/ApplyCoupon/ApplyCoupon.jsx]: The radio button itself is still interactable even if the purchase is below the coupon limit.

**Review by tanveertkd — APPROVED** [review]
Code is readable and easy to understand. Apart from the one tiny suggestion everything works as expected. 👍

File: src/Components/ApplyCoupon/ApplyCoupon.jsx (hunk 1 above)
Comment (BelideNikhil) [reply src/Components/ApplyCoupon/ApplyCoupon.jsx]: fixed it, thank you

**Pull request merged**